coralproject/talk
# Tracing

## What does this PR do?

Include trace ID in each downstream request.

## How do I test this PR?

Load the stream, look at the `/api/v1/graph/ql` request in the network inspector, notice the `X-Talk-Trace-ID` header that matches the `traceID` in the logs corresponding to the request.
